### ISSUE TITLE
Trivial Dirichlet character broken

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#5481]
This page gives a server error: https://beta.lmfdb.org/Character/Dirichlet/1/0.

**Comment by AndrewVSutherland** [comment]
Agreed, but how did you get to this page, is it linked to somewhere?

**Comment by roed314** [comment]
I just saw the error in the google cloud alerts.
